### PR TITLE
Add v5.0.1 changelog

### PR DESCRIPTION
**Pull request by kenhys**

### CODE REVIEW

**Comment by kenhys** [comment]
As this entry is a bit redundant, revise it.

**Comment by kenhys** [comment]
Now changelog.md was simplified.

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,18 @@
...
+
+### News
+
+**fluent-package v5.0.1 is a GA (General Availability) version of fluent-package v5 series.**
+
+In v5.0.1, minor bug and security related issue was fixed.
+
+* deb: cleanup /var/run correctly when removing `fluent-package`.
+* Update bundled protocol-http1 to 0.15.1 to reduce attack vector (HTTP
+  Request/Response smuggling vulnerability).
```
Comment (daipom) [inline CHANGELOG.md:15]: Should we also specify the CVE name `CVE-2023-38697 `?

Comment (kenhys) [reply CHANGELOG.md]: Thanks, I've updated.

**Review by daipom — APPROVED** [review]
LGTM! Thanks!

**Pull request merged**